sdk/servicebus/service-bus/src/util/concurrentExpiringMap.ts

```diff
@@ -22,6 +22,40 @@ export class ConcurrentExpiringMap<TKey> {
     this._delayBetweenCleanupInSeconds = options.delayBetweenCleanupInSeconds || 30;
   }
 
+  private async _scheduleCleanup(): Promise<void> {
```

Comment:
Is this due to the rule that all private functions should be before the public ones? I thought we wanted the other way around?